rosbag2_transport/src/rosbag2_transport/rosbag2_node.cpp

```diff
@@ -56,7 +56,9 @@ std::shared_ptr<GenericSubscription> Rosbag2Node::create_generic_subscription(
       topic,
       callback);
 
-    get_node_topics_interface()->add_subscription(subscription, nullptr);
+    // Intra-process message handling has to be disabled.
+    bool use_intra_process = false;
+    get_node_topics_interface()->add_subscription(subscription, nullptr, use_intra_process);
```

Comment:
I think this needs to be updated now, I removed the `use_intra_process` argument from this function.

Reply:
I fixed this and rebased.